backend/kernelCI/settings.py

```diff
@@ -193,43 +193,62 @@ def get_json_env_var(name, default):
 
 DATABASE_ROUTERS = ["kernelCI_app.routers.databaseRouter.DatabaseRouter"]
 
-DATABASES = {
-    "default": get_json_env_var(
-        "DB_DEFAULT",
-        {
-            "ENGINE": "django.db.backends.postgresql",
-            "NAME": "kernelci",
-            "USER": "kernelci",
-            "PASSWORD": "kernelci-db-password",
-            "HOST": "127.0.0.1",
-            "OPTIONS": {
-                "connect_timeout": 5,
-            },
-        },
-    ),
-    "dashboard_db": {
+# Database definition configurations
+kcidb_config = get_json_env_var(
+    "DB_DEFAULT",
+    {
         "ENGINE": "django.db.backends.postgresql",
-        "NAME": "dashboard",
-        "USER": get_json_env_var("DASH_DB_USER", "dev"),
-        "PASSWORD": get_json_env_var("DASH_DB_PASSWORD", "dev"),
-        "HOST": get_json_env_var("DASH_DB_HOST", "127.0.0.1"),
-        "PORT": get_json_env_var("DASH_DB_PORT", 5434),
+        "NAME": "kernelci",
+        "USER": "kernelci",
+        "PASSWORD": "kernelci-db-password",
+        "HOST": "127.0.0.1",
         "OPTIONS": {
             "connect_timeout": 5,
         },
     },
-    "cache": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": os.path.join(BACKEND_VOLUME_DIR, "cache.sqlite3"),
-    },
-    "notifications": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": os.path.join(BACKEND_VOLUME_DIR, "notifications.sqlite3"),
+)
+
+dashboard_db_config = {
+    "ENGINE": "django.db.backends.postgresql",
+    "NAME": "dashboard",
+    "USER": get_json_env_var("DASH_DB_USER", "dev"),
+    "PASSWORD": get_json_env_var("DASH_DB_PASSWORD", "dev"),
+    "HOST": get_json_env_var("DASH_DB_HOST", "127.0.0.1"),
+    "PORT": get_json_env_var("DASH_DB_PORT", 5434),
+    "OPTIONS": {
+        "connect_timeout": 5,
     },
 }
 
+# Common databases
+cache_db_config = {
+    "ENGINE": "django.db.backends.sqlite3",
+    "NAME": os.path.join(BACKEND_VOLUME_DIR, "cache.sqlite3"),
+}
+
+notifications_db_config = {
+    "ENGINE": "django.db.backends.sqlite3",
+    "NAME": os.path.join(BACKEND_VOLUME_DIR, "notifications.sqlite3"),
+}
+
+USE_NEW_DB = os.environ.get("USE_NEW_DB", False)
+if USE_NEW_DB is True:
```

Comment:
I think it is better this way:
```suggestion
USE_NEW_DB = os.environ.get("USE_NEW_DB", False) == "True"
if USE_NEW_DB:
```